inc/namespace.php

```diff
@@ -31,3 +33,42 @@ function on_plugins_loaded() {
 		require_once ROOT_DIR . '/vendor/johnbillion/query-monitor/query-monitor.php';
 	}
 }
+
+function qm_file_path_map( $map ) : array {
+	// Chassis and Local Server
+	if ( file_exists( '/etc/chassis-constants' ) ) {
+		$json_string = file_get_contents( '/etc/chassis-constants' );
+		$data = json_decode( $json_string, true );
+		if ( ! empty( $data['synced_folders']['/chassis'] ) ) {
+			$folder_path = $data['synced_folders']['/chassis'] . '/content/themes/base';
+			$map['/chassis/'] = $data['synced_folders']['/chassis'] . '/';
+		}
+	}
+
+	return $map;
+}
+
+function qm_file_link_format( $format ) : string {
```

Comment:
Needs a phpdoc block.

Reply:
Also, a more descriptive function name would be good. Something like `set_edit_link_format`?